### PR TITLE
DS-521 Support alignment prop on RadioButtonGroup

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**
I changed the spacing a bit: this matches additional changes made by Marcio

### CODE REVIEW

File: src/components/radio-button-group/RadioButtonGroup.tsx [hunk 1]
```diff
@@ -23,11 +23,17 @@ import * as RadioGroup from '@radix-ui/react-radio-group';
 import { ReactNode, forwardRef } from 'react';
 import { PropsWithLabels } from '~types/utils';
 
+export enum RadioButtonGroupAlignment {
```
Comment (david-cho-lerat-sonarsource) [inline src/components/radio-button-group/RadioButtonGroup.tsx:26]: don't this need to be re-exported in the index?

File: stories/RadioButtonGroup-stories.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 import type { Meta, StoryObj } from '@storybook/react';
 import { RadioButtonGroup } from '../src';
+import { RadioButtonGroupAlignment } from '../src/components/radio-button-group/RadioButtonGroup';
```
Comment (david-cho-lerat-sonarsource) [inline stories/RadioButtonGroup-stories.tsx:23]: RadioButtonGroupAlignment should be re-exported and imported on the same line as the one above

**Review by david-cho-lerat-sonarsource — CHANGES_REQUESTED** [review]
Missing a re-export, otherwise :+1:

**Comment by jeremy-davis-sonarsource** [comment]
> Missing a re-export, otherwise 👍

This was just to get a build, not yet ready for review :sweat_smile:

**Comment by david-cho-lerat-sonarsource** [comment]
> This was just to get a build, not yet ready for review 😅

oops, sorry, got too eager :sweat_smile:

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
great :+1:

**Pull request merged**